lib/src/firebase_cache_manager/web_firebase_cache_manager.dart

```diff
@@ -1,10 +1,12 @@
+import 'dart:io';
```

Comment:
Importing "dart:io" in this file will break the package's compatibility with Flutter Web.